docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst

```diff
@@ -0,0 +1,112 @@
+=================
```

Comment:
For this file to be included, you need to add a line in https://github.com/DIRACGrid/DIRAC/blob/integration/docs/source/AdministratorGuide/index.rst